readme.md

```diff
@@ -77,3 +77,45 @@ make call to `http://localhost:8999/jobs/create?job_type=add&job_parameters={%22
 ## Extending
 New jobs can be added by adding a new Celery function to the jobs folder and adding 
 the function to the jobs registry. See the ./packer/jobs/example.py to learn how.
+
+
+## Existing jobs
```

Comment:
there is also the `basic export` job - export of hypercube in a flat table format